pkg/ebpf/tracee.go

```diff
@@ -1034,6 +1038,9 @@ func (t *Tracee) initKsymTableRequiredSyms() error {
 			}
 		}
 	}
+
+	// bpf boot time helper
+	t.requiredKsyms = append(t.requiredKsyms, "bpf_ktime_get_boot_ns")
```

Comment:
I got you point, but it turns out a duplication as we already check it here: https://github.com/aquasecurity/tracee/blob/4b048001b82d4a0c8ac68943d077c95492852b6a/pkg/ebpf/tracee.go#L548

Reply:
I recall there was check doing this with the ksymbol table before, though maybe i'm mistaken. If this is the only check this change is redundant as you say.